README.md

```diff
@@ -9,12 +9,26 @@ We work hard to contribute to the Node.js community and we want to make sure you
 
 Note that this is not intended to be a comprehensive set of recommendations. Rather it's meant to be a helpful set of content that makes it easier to avoid any potential gotchas, and the beginning of what we expect to be an ongoing conversation on how we can improve the Node.js experience on Microsoft platforms.
 
-## Contribution guidelines
-
-### Pull requests are welcome!
-This is still a work in progress, an experiment - if you will, and we'd love your help to make it better! Please file an issue on any major changes you're planning - e.g. "Add new X section", and assign yourself to issues you intend to take on to prevent duplication of effort.
-
-It's also good to make a pull request for any change to the README (minor spelling and wording changes are fine to commit to master). This enables everyone to stay updated on what's going on in the repo. We'll be lax on this in the beginning as we work together to populate this with an initial set of content. 
+## Table of contents
+* [Hello World](docs/getting-started.md)
+* [Working with npm packages](docs/getting-started.md#working-with-npm-packages)
```

Comment:
Not a fan of imposing docs/* structure just yet, especially since it's not *really* docs... I don't think the hierarchy is necessary just yet, and would prefer to keep this set of content all at the root level for now